src/components/shared/IndividualCard.js

```diff
@@ -76,6 +76,14 @@ function renderField({ key, item, classes }) {
           <span className={classes.label}>Force Sensitive:</span> {item[key]}
         </Typography>
       )
+    case "price":
+      return (
+        <Typography key={key}>
+          <span className={classes.label}>Price:</span> {`${item.restricted ? "(R) " : ""}${item[key]}`}
```

Comment:
I should have been clearer in my last comment, but can you also humanize this exactly like it is in the table? i.e. `(R) 1,400`

Reply:
Oooh yeah, my bad. Forgot to add the .toLocaleString() at the end.